src/app/components/registration-dialog/registration-dialog.component.ts

```diff
@@ -47,10 +49,22 @@ export class RegistrationDialogComponent implements OnInit {
   }
 // Function try to save changes or create user to backend
   public trySave(user: any) {
-      this._user.createNewUser(user).subscribe(
+      this._user.signUp(user).subscribe(
         (resp: any) => {
-          console.log('resp', resp);
-          this.router.navigate([ '', 'home']);
+          this.authservice.authenticate(user).subscribe(
```

Comment:
No need to provide "authenticate" on "signUp". Reason: In signup response you already have all required info including "authToken" to add to localStorage. Remove additional request and do rest of logic inside signup response.